src/Pages/UserProfilePage.jsx

```diff
@@ -0,0 +1,40 @@
+import { React, useContext, useEffect } from 'react';
+import Store from '../Store/Store';
+import axios from 'axios';
+import Keys from '../config.keys';
+const UserProfilePage = () => {
+
+    const [userData, setUserData] = useContext(Store);
+    const isAuth = false;
+    useEffect(() => {
+        const getUserDetails = async () => {
+            console.log(userData);
+            try {
+                const url = Keys.BASE_API + '/user';
+                var res = await axios.put(url, {
```

Comment:
This ain't right. You are required to make a get request to fetch the registered events of the logged in user. Could you please check the Api docs once and change accordingly?